Lesson_5/Index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="icon" type="image/x-icon" href="CV-logo.png">
+    <title>Lesson 5- Resume</title>
+</head>
+
+<div>
```

Comment:
no div or any other block outside of `<body>`

Reply:
Changed